### PR TITLE
Only render content of selected Tab

### PR DESCRIPTION
**Pull request by AlanGreene**
# Changes

By default Carbon will render and hide the content of all Tab
components in a Tabs component.

Create a proxy Tabs component that provides a custom `renderContent`
prop to all child Tab components ensuring their content is only
rendered when the tab is selected.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: src/containers/TriggerTemplate/TriggerTemplate.test.js [hunk 1]
```diff
@@ -417,16 +417,12 @@ it('TriggerTemplateContainer renders full resource template information', async
   const compareToName = resourceTemplate1Details.metadata.name;
   await waitForElement(() => getByText(compareToName));
   await waitForElement(() => getByText(/revision/i));
-  await waitForElement(() => getByText(/params.gitrevision/i));
```
Comment (steveodonovan) [inline src/containers/TriggerTemplate/TriggerTemplate.test.js]: why are these removed ?

Comment (AlanGreene) [reply src/containers/TriggerTemplate/TriggerTemplate.test.js]: because they were only rendered in the inactive tab content which is no longer present, they shouldn't have been included in the test in the first place
